balancer-js/src/pool-composable-stable/encoder.ts

```diff
@@ -11,7 +11,7 @@ export enum ComposableStablePoolJoinKind {
 export enum ComposableStablePoolExitKind {
   EXACT_BPT_IN_FOR_ONE_TOKEN_OUT = 0,
   BPT_IN_FOR_EXACT_TOKENS_OUT,
-  EXACT_BPT_IN_FOR_ALL_TOKENS_OUT,
+  EXACT_BPT_IN_FOR_TOKENS_OUT,
```

Comment:
This is in fact following the monorepo encoder. Just letting you know that changing this will break consistency between monorepo and sdk encoders, even though I'm not sure if that's actually a requirement.
Besides that, I'd double check if there's even need for a `ComposableStablePoolEncoder` at all. Unless I'm missing something, it looks the same as `StablePoolEncoder`. Maybe a better refactor would be to remove/reuse.

Reply:
Probably best to keep consistency then, just to make it easier to compare.

Reply:
sure, removed the change